### ISSUE TITLE
[Feature request] Avoid double import of GPX-files

### ISSUE THREAD

**Issue by update-freak** [Flomp/wanderer#139]
Hi developer,

I import all my GPX-file (about 200) from a folder.
Accidently I imported them again after I did the update to v.0.12.0.
So maybe it would be good to have a import check to avoid double import, e.g. depending on name etc. a message:
"Track XY already exists. Are you sure to import this track" -> Yes, to still import and No, to skip the import of this single file.
This should then also work for importing a whole folder with multiple files.

**Comment by update-freak** [comment]
and I think it would makes to have also the following option in the settings how to handle the import of double tracks:

- skip all double found tracks while the import => with this setting it's still possible to select whole folders with large amount of GPX-files
- raise the message while importing and a track is already there => message as in my first post (skip or import)

**Comment by Flomp** [comment]
I agree that this should be an option when importing. IMO, this check should not simply be based on the name though but rather on geospatial similarity between tracks (aka the uploaded track follows an already uploaded track closely). 

That is of course a much more complicated problem to solve than a simple string comparison. But I think that this is a feature that might also come in handy in other parts of the app. So I'll look into it and report back.

**Comment by githubhacs** [comment]
Just to add to this.

"geospatial similarity between tracks" - as long as its also comparing the date/time stamps for the track?

Many of us will have the 'same' tracks but recorded at different times, which could be days, weeks, months or years apart.

**Comment by Flomp** [comment]
v0.15.0 comes with duplicate detection. It's based on bounding box, centroid, start & end position of the track. You can still force the upload if you want.
